docs/releases/v0_4_0.md

```diff
@@ -3,7 +3,7 @@
 ## Summary
 Although the focus of this release was on enabling SPDX formatted reports, a good chunk of work went into resolving technical debt. Most notably Michael Rohan's work towards cleaning up code style and enabling a workflow for contributors to check for linting errors. There was also a lot of work by Santiago Torres towards making Tern pip installable.
 
-Tern now supports the SPDX document format. As of this release the documents produced are valid SPDX. You can try it out by running `tern -l report -i <docker image> -m spdxtagvalue -f spdx-container-image.txt`. You can validate the document using [SPDX tools](https://github.com/spdx/tools). Note that python tools are still a work in progress but you can get on the [mailing list](https://lists.spdx.org/g/spdx-tech) and ask questions about the document there.
```

Comment:
You don't have to modify the command here in the releases. The `-l` command will still work here.

Reply:
fixed